pyblish_lite/window.py

```diff
@@ -830,6 +831,17 @@ def on_was_processed(self, result):
             if instance.id not in models["instances"].ids:
                 models["instances"].append(instance)
 
+            family = instance.data['family']
+            if family:
+                plugins_filter = self.data['views']['right'].model()
+                plugins_filter.add_inclusion(role='families', value=family)
+
+            families = instance.data['families'] if 'families' in instance.data  else False
```

Comment:
Or even..

```python
families = instance.data.get('families')
```

As the default value is `None` which would pass the `if`-test below.